scripts/docs_changelog_generator/utils.py

```diff
@@ -0,0 +1,45 @@
+import unicodedata
+from pathlib import Path
+
+from gitchangelog.gitchangelog import GitCommit, GitRepos
+
+DEFAULT_REPO_URL = "https://github.com/IBM/ibm-generative-ai"
+
+
+def output_engine_pipeline(render_fn):
+    def _process(output_engine):
+        return lambda data, opts: output_engine(*render_fn(data, opts))
+
+    return _process
```

Comment:
```python
output_engine_pipeline = lambda render_fn: lambda output_engine: lambda data, opts: output_engine(*render_fn(data, opts))
```

😄

Reply:
noooo 🙈 :D 
I add more code just to prevent this :D